frontend/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "sharespots",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "scripts": {
+    "build-watch": "tsc -w",
+    "build": "tsc",
+    "tslint": "tslint --project tsconfig.json --format stylish --exclude **/src/**/*.js",
```

Comment:
We use TypeScript in this project, all files should be `.ts` instead of `.js` this will allow us to use normal javaScript but get the linting power and type checking of TypeScript. Don't worry about it yet, there is nothing more that is needed to be known about it at this stage.